instrumentation/messaging/src/main/java/brave/messaging/MessagingRequest.java

```diff
@@ -59,13 +57,26 @@
   /**
    * Messaging channel name, e.g. "hooks" or "complaints". {@code null} if unreadable.
    *
-   * <p>Conventionally associated with the key "messaging.channel_name"
```

Comment:
ps sometime around july last year we decided to replace message.X with messaging.X I don't recall the rationale but it should have been written down. I think it was to do with how the other properties would sound if prefixed "message" also to match the abstraction name.

The only reason I mention this is that we should decide if the tag name "messaging.id" is ok for the message id or we make "messaging.message_id" which is also awkward for the same. I don't have a strong feeling either way, but happy to add to the RATIONALE whatever we choose.

Reply:
Lean towards `messaging.id` since when in doubt, shorter seems better